Rakefile

```diff
@@ -1,5 +1,7 @@
+$:.unshift "lib"
+$:.unshift "test"
```

Comment:
You might want to use absolute paths (possibly derived from the location of the `Rakefile`) instead. Depending on when things are `require`d in the code, otherwise an innocent looking `cd` might unexpectedly break things.